src/handlers/motions/motionFinalized/motionFinalized.ts

```diff
@@ -60,10 +63,19 @@ export default async (event: ContractEvent): Promise<void> => {
       ),
     );
 
+    const updatedMessages = [
+      ...messages,
+      {
+        name: MotionEvents.MotionFinalized,
+        messageKey: `${transactionHash}${logIndex}`,
```

Comment:
This formatting seems to be repeated in a few places. You could consider adding a util function to generate it

Reply:
@jakubcolony you mean like this?
`export const getMessageKey = (
  transactionHash: string,
  logIndex: number,
): string => {
  return `${transactionHash}${logIndex}`;
};`